loleaflet/src/core/Socket.js

```diff
@@ -1018,6 +1021,21 @@ app.definitions.Socket = L.Class.extend({
 			var mobileEvent = textMsg.substring('mobile: '.length);
 			this._map.fire(mobileEvent);
 		}
+		else if (textMsg.startsWith('blockui: ')) {
+			textMsg = textMsg.substring('blockui: '.length);
+			msg = null;
+
+			if (textMsg === 'rename') {
+				msg = _('The document is being renamed and will reload shortly');
+			}
+
+			this._map.fire('blockUI', {message: msg});
+			return;
+		}
+		else if (textMsg.startsWith('unblockui: ')) {
```

Comment:
Since the message is optional (even for blockui, but certainly for unblockui, which will almost never have a message), we shouldn't expect the trailing space.

I suggest we do the startsWith for both 'blockui:' and 'unblockui:' *without* the trailing space.

Reply:
done, fixed in both PR